packages/contracts/src/dollar/core/CreditClock.sol

```diff
@@ -73,7 +73,7 @@ contract CreditClock {
             if (blockNumber < block.number)
                 revert("CreditClock: block number must not be in the past.");
         }
-
+        // slither-disable-next-line divide-before-multiply
```

Comment:
This seems like a bad idea?

Reply:
the multiply is done through abdk so overflow underflow are handled and it is floating point so no problem there

Reply:
> This seems like a bad idea?

Slither isn't perfect and might throw false positive or couldn't understand the use cases.